docs/reference/nodejs/v0/api/api.md

```diff
@@ -71,6 +71,19 @@ import { authMiddleware } from '../middleware';
 const privateApi = api('private', { middleware: authMiddleware });
 ```
 
+### Define middleware
+
+```javascript
+const authMiddleware = (ctx, next) => {
+  // Perform auth validation.
+  next(ctx);
+};
+```
```

Comment:
should this example also be async like the python examples?

```suggestion
```javascript
const authMiddleware = async (ctx, next) => {
  // Perform auth validation.
  return await next(ctx);
};
```
```